src/ui/user-profile/common/UserProfileForm.js

```diff
@@ -108,6 +86,64 @@ export class UserProfileFormBody extends Component {
     this.props.onWillUnmount();
   }
 
+  generatetValidatorFunc(value, validatorFuncName, validatorFunc, validatorArray, parseValueFunc) {
```

Comment:
`generatetValidatorFunc` => `generateValidatorFunc`

On top of that, there's room for optimization regarding this method.